config/database.yml

```diff
@@ -1,27 +1,53 @@
+<%
+  # first env with a value
+  def first_of(*envs)
+    envs.map { |env| ENV[env] }.compact.first
+  end
+
+  def puts_red(msg)
+    puts "\e[31m#{msg}\e[0m"
+  end
+
+  def deprecated(env, message)
+    puts_red "DEPRECATED: #{env} is deprecated, #{message}" if ENV[env]
+  end
+
+
+  deprecated("HOSTEDGPT_DATABASE_PORT", "use HOSTEDGPT_DB_PORT instead")
+  deprecated("HOSTEDGPT_DEV_DB", "use HOSTEDGPT_DB_NAME instead")
+  deprecated("HOSTEDGPT_TEST_DB", "provide HOSTEDGPT_DB_NAME and test database name is generated")
+  deprecated("HOSTEDGPT_DATABASE_PASSWORD", "use HOSTEDGPT_DB_PASSWORD instead")
```

Comment:
I wish there was a helper file for this. I guess I could put this in lib and include it manually. Maybe this could be an initializer too, but I wanted to keep this close to where the env vars are used. Or if we remove other envs in the future then maybe it's better in an initializer